<ui version="4.0" >
 <class>ProgressDialog</class>
 <widget class="QDialog" name="ProgressDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>222</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="cancelButton" >
   <property name="geometry" >
    <rect>
     <x>130</x>
     <y>170</y>
     <width>123</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text" >
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar" >
   <property name="geometry" >
    <rect>
     <x>30</x>
     <y>30</y>
     <width>341</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value" >
    <number>24</number>
   </property>
  </widget>
  <widget class="QLabel" name="label" >
   <property name="geometry" >
    <rect>
     <x>180</x>
     <y>70</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text" >
    <string>Tracks</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2" >
   <property name="geometry" >
    <rect>
     <x>180</x>
     <y>100</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text" >
    <string>Log Points </string>
   </property>
  </widget>
  <widget class="QLabel" name="m_num_tracks" >
   <property name="geometry" >
    <rect>
     <x>90</x>
     <y>70</y>
     <width>75</width>
     <height>22</height>
    </rect>
   </property>
   <property name="layoutDirection" >
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text" >
    <string>0</string>
   </property>
   <property name="scaledContents" >
    <bool>false</bool>
   </property>
   <property name="alignment" >
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="m_num_logpoints" >
   <property name="geometry" >
    <rect>
     <x>90</x>
     <y>100</y>
     <width>75</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text" >
    <string>0</string>
   </property>
   <property name="alignment" >
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
